### ISSUE TITLE
May I know how to get all available pure backbone model ?

### ISSUE THREAD

**Issue by HandsomeBrotherShuaiLi** [qubvel/segmentation_models#519]
Hi,
Thanks for your great work and it helps me a lot for the segmentation tasks. I find that there are so many implemented classic backbone models in your framework. 
So may I know how to import these great backbones in our own script via segmentation_models lib?
For instance:
`from segmentation_models.backbones import resnet18` 
etc.

Thanks ~

**Comment by mehdi2020** [comment]
model = Unet(backbone_name='resnet18', ...)
https://segmentation-models.readthedocs.io/en/latest/tutorial.html

**Comment by HandsomeBrotherShuaiLi** [comment]
> Hi, you can write: model = Unet(backbone_name='resnet18', ...) https://segmentation-models.readthedocs.io/en/latest/tutorial.html

Hi, thanks for your reply. But my purpose is that I don't want encoder-decoder framework, I just want borrow all classification backbone models from this library without any decoder, that is to say, I only want the downsampling encoder model without upsampling decoder.

**Comment by HandsomeBrotherShuaiLi** [comment]
I wrote some ugly codes to implement my idea:
`import segmentation_models as sm

sm.set_framework('tf.keras')
import tensorflow.keras as keras
keras.backend.set_image_data_format('channels_last')
backbone = "vgg16"
framework = sm.Unet(backend=backbone,  
                    encoder_weights='imagenet',
                    activation="sigmoid",
                    classes=1,
                    input_shape=(256, 256, 3))
encoder_features = sm.Backbones.get_feature_layers(backbone.lower(), n=4)
final_downsampling_feature = framework.get_layer(name=encoder_features[0]).output
model = keras.models.Model(framework.input, final_downsampling_feature)
model.summary()`
